src/Components/People.js

```diff
@@ -0,0 +1,60 @@
+import { Component } from 'react';
+import axios from 'axios';
+
+class People extends Component {
+    state = {
+        searchPerson: '',
+        person: {},
```

Comment:
It seems like you're render method expects this to be an array, not an object. Consider setting its initial value to an empty array!